src/Controls/Maps/src/AppHostBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Maui.Controls.Maps;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Maps.Handlers;
+
+namespace Microsoft.Maui.Controls.Hosting
+{
+	public static partial class AppHostBuilderExtensions
+	{
+#if __ANDROID__ || __IOS__
+		public static IMauiHandlersCollection AddMauiMapsControlsHandlers(this IMauiHandlersCollection handlersCollection)
```

Comment:
I would add a `UseMauiMaps` call that's available on all platforms so users don't have to if/def to access it.

Reply:
Done